src/components/MessagingWindow/index.jsx

```diff
@@ -34,9 +32,9 @@ class MessagingWindow extends Component {
   handleClick = e => {
     e.preventDefault();
 
-    const { text, user, receiver } = this.state;
+    const { text, user } = this.state;
     const message = { text, user_id: user };
-    firebaseWrapper.sendMessage(user, receiver, message);
+    firebaseWrapper.sendMessage("asdf", user, message);

```

Comment:
wait the way you're calling this doesn't make sense because the interface is `(conversationId, message)`